docs/source/usage/workflows/memoryPerDevice.py

```diff
@@ -4,11 +4,12 @@
 This file is part of PIConGPU.
 
 Copyright 2018-2019 PIConGPU contributors
-Authors: Marco Garten
+Authors: Marco Garten, Paweł Ordyna
```

Comment:
Just a note: for files consumed by compilers in PIConGPU (C++ code), please try to use ASCII only since some older compilers and systems are not too flexible :-S
For Python code, this is fine though.